docs/specification/current.md

```diff
@@ -982,8 +982,8 @@ The **Coding** data type captures the concept of a code. A code is understood on
 Field | Optionality | Type | Description
 ----- | ----- | ----- | --------
 `code` | REQUIRED | *string* | The code for what is being represented
-`system` | OPTIONAL | *string* | A codesystem for this `code`.
-`display` | OPTIONAL | *string* | A short, human-readable label to display.
+`system` | REQUIRED | *string* | A codesystem for this `code`.
+`display` | CONDITIONAL | *string* | A short, human-readable label to display. REQUIRED for [Override Reasons](#overridereason), OPTIONAL for [Topic](#topic).
```

Comment:
Rather than list use-cases within the common structure, it may be better to have a general statement deferring to consuming use-cases.  Then, within override reasons (both instances) and topic, list whether a display is required.

Reply:
* card.overrideReasons: https://cds-hooks.org/specification/current/#card-attributes
 * .display is required. Added normative language.
* feedback.overrideReasons: https://cds-hooks.org/specification/current/#feedback
 * .display is optional; therefore, don't need to re-iterate base definition (did add link to base coding definition).
* card.source.topic: https://cds-hooks.org/specification/current/#source
 * .display is optionaltherefore, don't need to re-iterate base definition (did add link to base coding definition).